### PR TITLE
IASC-761 Google search

### PR DESCRIPTION
**Pull request by left23**
Refs: IASC-761 and IASC-764 and IASC-765

~~Paragraph type with ID field for Google Custom Search Engine.~~
~~Paragraph template override includes container divs, attaches the library for custom CSS, and adds the scripts.~~

The global search form template override includes a change to the name value to match the default GCSE name. 

Removal of Solr, use Search API for DB

~~GCSE used for AAP Services directory search~~

Download all Services button added

Sitemap installed (already on Prod https://github.com/UN-OCHA/iasc8/pull/726)

IASC-767 Remove Search block template override in custom module

Install `ocha_search` module, configure, place block and adjust sub theme

### CODE REVIEW

File: html/themes/custom/iasc_common_design/templates/cd/cd-header/cd-search.html.twig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+<div class="cd-search">
+
+  {{ attach_library('iasc_common_design/google-cse') }}
+
+  <div class="search-block-form cd-search__form block block-search" data-drupal-selector="search-block-form" aria-labelledby="cd-search-form" data-cd-toggable="Search" data-cd-component="cd-search" data-cd-logo="search" data-cd-focus-target="cd-search" id="block-iasc-common-design-searchform" role="search">
+
+    {# BEGIN Google embed code #}
+    <script async src="https://cse.google.com/cse.js?cx=077aef2e37d404230"></script>
```
Comment (attiks) [inline html/themes/custom/iasc_common_design/templates/cd/cd-header/cd-search.html.twig:9]: can you add `defer` as well

Comment (left23) [reply html/themes/custom/iasc_common_design/templates/cd/cd-header/cd-search.html.twig]: Yup!

**Review by attiks — APPROVED** [review]

**Comment by left23** [comment]
@attiks This is ready for another review. I'm still determining the best way to do this so feel free to suggest better ways to tackle this. The node template override with the specific Node ID isn't the proper way to do this, but it demonstrates what I am going for.

**Comment by left23** [comment]
@attiks Do we keep this synced with `develop` or are we specifically not including the sitemap from https://github.com/UN-OCHA/iasc8/pull/726 on this feature branch?

**Comment by attiks** [comment]
dev merged

**Pull request merged**